examples/cc3220/README.md

```diff
@@ -0,0 +1,134 @@
+Installing Dependencies & Related Documentation
```

Comment:
Should we take this time to move this to /examples/cc3200sf/README.md ?  I think this will only work with the SF, correct?